### PR TITLE
Made a fix

### PR DESCRIPTION
**Pull request by dalvira**
spelling corrections

### CODE REVIEW

File: exercises/01-SpellCheck.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Greatness ensues
 
-Developer peeps,
+Developer people,
```
Comment (caabernathy) [inline exercises/01-SpellCheck.md:4]: Very nice fix.

**Review by caabernathy — COMMENTED** [review]
I like this.

**Comment by joseph-ortiz** [comment]
👍

**Comment by danalamo** [comment]
` Null `

**Pull request merged**